snpeff/5.1/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:focal as app
+
+ARG SNPEFF_VER="5.1"
+
+# Metadata
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="SnpEff & SnpSift"
+LABEL software.version="5.1"
+LABEL description.SnpEff="Genetic variant annotation and effect prediction toolbox."
+LABEL description.SnpSift="Used after SnpEff annotation to filter and manipulate annotated files."
+LABEL website="https://pcingola.github.io/SnpEff"
+LABEL license="https://github.com/pcingola/SnpEff/blob/master/LICENSE.md"
+LABEL maintainer="Tom Iverson"
+LABEL maintainer.email="tiverson@utah.gov"
+
+# Install open using apt 
+RUN apt-get update && apt-get install -y software-properties-common &&\
+  add-apt-repository -y ppa:openjdk-r/ppa &&\
```

Comment:
I'm not 100% sure, but I don't think you need this line for `add-apt-repository -y ppa:openjdk-r/ppa &&\`

openjdk-11-jre should be readily available via `apt` for ubuntu:focal https://packages.ubuntu.com/focal/openjdk-11-jre

Reply:
You're right. It's available and installs fine. Thanks!